src/Microsoft.AspNetCore.Sockets.Client/WebSocketsTransport.cs

```diff
@@ -56,21 +57,26 @@ public WebSocketsTransport(ILoggerFactory loggerFactory)
         {
             while (!cancellationToken.IsCancellationRequested)
             {
-                const int bufferSize = 1024;
+                const int bufferSize = 1;
```

Comment:
I guess it's a leftover... 

I suggest you add a test where you send small message (e.g. 17 bytes), medium message (e.g. 11Kb) and a large-ish message (e.g. 7 MB)